bundles/pom.xml

```diff
@@ -5,649 +5,1227 @@
+
     <module>org.openhab.binding.solaredge</module>
+
     <module>org.openhab.binding.solarlog</module>
     <module>org.openhab.binding.solarmax</module>
     <module>org.openhab.binding.solarwatt</module>
+
     <module>org.openhab.binding.somfymylink</module>
+
     <module>org.openhab.binding.somfytahoma</module>
     <module>org.openhab.binding.somneo</module>
     <module>org.openhab.binding.sonnen</module>
+
     <module>org.openhab.binding.sonos</module>
+
     <module>org.openhab.binding.sonyaudio</module>
+
     <module>org.openhab.binding.sonyprojector</module>
+
     <module>org.openhab.binding.souliss</module>
+
     <module>org.openhab.binding.spotify</module>
+
     <module>org.openhab.binding.squeezebox</module>
+
     <module>org.openhab.binding.surepetcare</module>
+
     <module>org.openhab.binding.synopanalyzer</module>
+
     <module>org.openhab.binding.systeminfo</module>
+
     <module>org.openhab.binding.tacmi</module>
+
     <module>org.openhab.binding.tado</module>
+
     <module>org.openhab.binding.tankerkoenig</module>
+
     <module>org.openhab.binding.tapocontrol</module>
+
     <module>org.openhab.binding.telegram</module>
+
     <module>org.openhab.binding.teleinfo</module>
+
     <module>org.openhab.binding.tellstick</module>
+
     <module>org.openhab.binding.tesla</module>
+
     <module>org.openhab.binding.tibber</module>
+
     <module>org.openhab.binding.tivo</module>
+
     <module>org.openhab.binding.touchwand</module>
+
     <module>org.openhab.binding.tplinksmarthome</module>
+
     <module>org.openhab.binding.tr064</module>
+
     <module>org.openhab.binding.tradfri</module>
+
     <module>org.openhab.binding.twitter</module>
+
     <module>org.openhab.binding.unifi</module>
+
     <module>org.openhab.binding.unifiedremote</module>
+
     <module>org.openhab.binding.upnpcontrol</module>
+
     <module>org.openhab.binding.upb</module>
+
     <module>org.openhab.binding.urtsi</module>
+
     <module>org.openhab.binding.valloxmv</module>
+
     <module>org.openhab.binding.vdr</module>
+
     <module>org.openhab.binding.vektiva</module>
+
     <module>org.openhab.binding.velbus</module>
+
     <module>org.openhab.binding.velux</module>
+
     <module>org.openhab.binding.venstarthermostat</module>
+
     <module>org.openhab.binding.ventaair</module>
+
     <module>org.openhab.binding.verisure</module>
     <module>org.openhab.binding.vesync</module>
     <module>org.openhab.binding.vigicrues</module>
+
     <module>org.openhab.binding.vitotronic</module>
+
     <module>org.openhab.binding.volvooncall</module>
+
     <module>org.openhab.binding.warmup</module>
+
     <module>org.openhab.binding.weathercompany</module>
+
     <module>org.openhab.binding.weatherunderground</module>
+    <module>org.openhab.binding.webexteams</module>
+
     <module>org.openhab.binding.webthing</module>
+
     <module>org.openhab.binding.wemo</module>
+
     <module>org.openhab.binding.wifiled</module>
```

Comment:
Can you remove all added new lines so only `<module>org.openhab.binding.webthing</module>` is added?
This is issue #13405. :confused:

Reply:
Done.